src/main/webapp/WEB-INF/jsp/Main.jsp

```diff
@@ -25,14 +25,14 @@
   </header>
   <main>
     <section class="whatToDo">
-      <h1>I want to ...</h1>
+      <h1>Mig langar að ...</h1>
     </section>
     <section class="add" id="rider">
         <a></a>
     </section>
     <section class="pickContainer">
-      <h5 class="selectRider">Ride</h5>
-      <h5 class="selectDriver">Drive</h5>
+      <h5 class="selectRider">Bjóða ísFar</h5>
+      <h5 class="selectDriver">Fá ísFar</h5>
```

Comment:
translating to Icelandic, as 95-100% of users speaks icelandic,
we will probably add later translations for english as well, but until then, I am changing this to Icelandic,
If you can find a better copy, please change it :)